src/mono/mono/mini/mini-wasm-debugger.c

```diff
@@ -53,6 +53,7 @@ EMSCRIPTEN_KEEPALIVE gboolean mono_wasm_get_array_values (int object_id, int sta
 EMSCRIPTEN_KEEPALIVE gboolean mono_wasm_invoke_getter_on_object (int object_id, const char* name);
 EMSCRIPTEN_KEEPALIVE gboolean mono_wasm_invoke_getter_on_value (void *value, MonoClass *klass, const char *name);
 EMSCRIPTEN_KEEPALIVE gboolean mono_wasm_get_deref_ptr_value (void *value_addr, MonoClass *klass);
+EMSCRIPTEN_KEEPALIVE void mono_wasm_set_is_debugger_attached (gboolean is_attached);
```

Comment:
Is there a reason you have to forward declare this?

Reply:
No, should I remove?

Reply:
It doesn't harm anything, but I saw it's been done on a bunch of functions in this file that aren't called before the corresponding implementation and was wondering if I was missing something.

Reply:
Ah no, there does seem to be a reason... Those are wrapped in `G_BEGIN_DECLS`, so I guess they exist so this correctly builds as C++ or something? Maybe someone else remembers what was the motivation for this.